src/FwdState.cc

```diff
@@ -1508,6 +1508,12 @@ getOutgoingAddress(HttpRequest * request, const Comm::ConnectionPointer &conn)
         // else no tproxy today ...
     }
 
+    if(Config.onoff.transparent_outgoing_ip) {
+        conn->local = request->clientConnectionManager->clientConnection->local;
```

Comment:
This will crash on Squid generated requests when `request->clientConnectionManager` or `request->clientConnectionManager->clientConnection` are unset.